shared/src/iosMain/kotlin/io.newm.shared/internal/implementations/PreferencesDataStoreImpl.ios.kt.kt

```diff
@@ -0,0 +1,39 @@
+package io.newm.shared.internal.implementations
+
+import io.newm.shared.internal.db.PreferencesDataStore
+import platform.Foundation.NSUserDefaults
+
+class PreferencesDataStoreImpl : PreferencesDataStore {
```

Comment:
@marty can you double check this works well on IOS? I don't have a physical device to install the iOS app in